pkgs/by-name/fi/fido2luks/package.nix

```diff
@@ -16,17 +16,32 @@ rustPlatform.buildRustPackage rec {
     sha256 = "sha256-bXwaFiRHURvS5KtTqIj+3GlGNbEulDgMDP51ZiO1w9o=";
   };
 
-  nativeBuildInputs = [ pkg-config rustPlatform.bindgenHook ];
+  postPatch = ''
+    rm Cargo.toml Cargo.lock
+    ln -s ${./Cargo.toml} Cargo.toml
+    ln -s ${./Cargo.lock} Cargo.lock
```

Comment:
Whats the reason why we delete both files? Can't we work with patches?

Reply:
I used patches at first, but since we have to vendor the Cargo.toml anyway due to git dependencies, I thought this to be the simpler solution.